client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -38,7 +39,20 @@ export default {
       };
     }
   },
-
+  mounted() {
+    //Adjusts the size to fix the geoMap display if any of the contents sizes change.
+    if (this.$refs.container) {
+      this.observer = new ResizeObserver(() => {
+        this.onResize();
+      });
+      this.observer.observe(this.$refs.container);
+    }
+  },
+  beforeDestroy() {
+    if (this.observer) {
+      this.observer.unobserve(this.$refs.container);
+    }
+  },
```

Comment:
Now that you're using the more robust `ResizeObserver` should the `v-resize` event from Vuetify be dropped from the template?

Reply:
thank you, don't know why I glossed over the fact that `annotator` was a mixin for each of these components.